missal1962/ical.py

```diff
@@ -22,6 +22,39 @@ def build(days: Dict[datetime.date, Day], rank: int, lang: str) -> str:
             event.add("dtstart", datetime_)
             event.add("dtstamp", now)
             event.add("uid", f"{datetime_.strftime('%Y%m%d')}@missalemeum.com")
-            event.add("description", "https://www.missalemeum.com/{}/{}".format(lang, datetime_.strftime("%Y-%m-%d")))
+
+            # Map color arrau to an array of human readable strings
+            color_map = {
+              "w": "white",
+              "r": "red",
+              "b": "black",
+              "v" : "violet",
+              "g" : "green",
```

Comment:
As the application supports multiple languages, these names must be internationalized. See the example here https://github.com/mmolenda/Missal1962/blob/master/missal1962/views.py#L151. More details are in readme, in section Localization. For now there are two language versions. You can put English names to Polish files and I'll make the translation.

Reply:
Also, please make the map a class variable as there's no point to initialize it on each request.

Reply:
Thanks! I'll begin to get these fixes made.